easybuild/easyconfigs/n/NAMD/NAMD-2.12-intel-2017a-mpi.eb

```diff
@@ -9,10 +9,14 @@ description = """NAMD is a parallel molecular dynamics code designed for high-pe
 toolchain = {'name': 'intel', 'version': '2017a'}
 toolchainopts = {'pic': True, 'usempi': True}
 
-sources = ['NAMD_%(version)s_Source.tar.gz']
+sources = [{
+    'filename': 'NAMD_%(version)s_Source.tar.gz',
+    'extract_cmd': "tar xfv %s",  # source file is actually not gzipped
+}]
+checksums = ['3be34f04c64a1e4225c04560c676ca8ff9bb57aa8830e42a46bf9ac6b16eb403']
 
 dependencies = [
-    ('Tcl', '8.6.6'),
+    ('Tcl', '8.6.7'),
```

Comment:
@tovrstra Don't bump dependency versions in existing easyconfigs, since that may cause problems elsewhere (e.g. version conflicts on Tcl when this NAMD is reinstalled and then combined with existing modules).

Also, there's no easyconfig file for Tcl 8.6.7 that is compatible with `inte/2017a`.